charts/gundeck/templates/redis-ca-secret.yaml

```diff
@@ -0,0 +1,30 @@
+---
+{{- if not (empty .Values.config.redis.tlsCa) }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "gundeck-redis-ca"
+  labels:
+    app: gundeck
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  ca.pem: {{ .Values.redis.tlsCa | b64enc | quote }}
```

Comment:
**issue:** This leads to a NPE when `config.redis.tlsCa` is set.


**suggestion:**
```suggestion
  ca.pem: {{ .Values.config.redis.tlsCa | b64enc | quote }}
```